source/docs/zero-to-robot/step-2/wpilib-setup.rst

```diff
@@ -59,38 +61,43 @@ Upon opening the installer, you'll be presented with the below screen. Go ahead
 .. image:: images/wpilib-setup/installer-start.png
    :alt: Start of Installer
 
+.. image:: images/wpilib-setup/installer-options.png
+   :alt: An overview of the installer options
+
+This showcases a list of options included with the WPILib installation.
+
+- :guilabel:`Tools Only` installs just the WPILib tools (Pathweaver, Shuffleboard, RobotBuilder, SysID, Glass, and OutlineViewer) and JDK.
+- :guilabel:`Everything` installs the full development environment (VS Code, extensions, all dependencies), WPILib tools, and JDK.
+
+You will notice two buttons, :guilabel:`Install for this User` and :guilabel:`Install for all Users`. :guilabel:`Install for this User` only installs it on the current user account, and does not require administrator privileges. However, :guilabel:`Install for all Users` installs the tools for all system accounts and *will* require administrator access. :guilabel:`Install for all Users` is not an option for macOS and Linux.
+
+.. note:: If you select Install for all Users, Windows will prompt for administrator access through UAC during installation.
+
+Select the option that is appropriate for you, and you'll presented with the following installation screen.
+
 This next screen involves downloading VS Code. Unfortunately, due to licensing reasons, VS Code can not be bundled with the installer.
 
 .. image:: images/wpilib-setup/installer-vscode-download.png
    :alt: Overview of VS Code download options
 
-- Download VS Code for Single Install
+- Download for this computer only
 
   - This downloads VS Code only for the current platform, which is also the smallest download.
 
-- Skip installing VS Code
+- Skip and don't use VS Code
 
   - Skips installing VS Code. Useful for advanced installations or configurations. Generally not recommended.
 
-- Use Downloaded Offline Installer
+- Select existing VS Code zip for offline install on this computer
 
   - Selecting this option will bring up a prompt allowing you to select a pre-existing zip file of VS Code that has been downloaded by the installer previously. This option does **not** let you select an already installed copy of VS Code on your machine.
 
-- Download VS Code for Offline Install
+- Create VS Code zip to share with other computers/OSes for offline install
```

Comment:
Is this option really that long?

Reply:
See the picture

https://github.com/wpilibsuite/WPILibInstaller-Avalonia/pull/209